code/modules/mob/living/simple_animal/friendly/raven.dm

```diff
@@ -0,0 +1,26 @@
+/mob/living/simple_animal/pet/raven
+	name = "raven"
+	desc = "Quoth the raven nevermore! Just now in space!"
+	icon = 'icons/mob/animal.dmi'
+	icon_state = "raven"
+	icon_living = "raven"
+	icon_dead = "raven_dead"
+	speak_emote = list("caws","caw")
+	health = 7
+	maxHealth = 7
+	attacktext = "pecks violently"
```

Comment:
This also doesn't make much sense in a sentence to me.

Reply:
Give me a suggestion maybe? I think pecks is a pretty understandable attack reaction for a bird? I changed it to bites in response.

Reply:
pecks by itself is fine, just that it'll say `raven pecks violently [name]` if it were like that, which doesn't make much sense to me.

Reply:
Oh I get it.